test/scripts/jenkins_xpack_package_deb.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -e
+
+source src/dev/ci_setup/setup_env.sh
+
+gsutil -q -m cp "gs://ci-artifacts.kibana.dev/package-testing/$GIT_COMMIT/kibana-*.deb" ./target
+
+export VAGRANT_CWD=test/package
+vagrant up deb --no-provision
+
+node scripts/es snapshot \
```

Comment:
My only concern here is that ES will never be stopped.

Reply:
Not super concerned since these machines are only used for one job, but if this moves to Buildkite, we will need to make sure that either the script cleans up the ES instance (`trap` should be good enough) or the agent type in Buildkite is a single-use machine.